modules/data.atmosphere/R/download.ECMWF.R

```diff
@@ -0,0 +1,129 @@
+#' Download ECMWF Open Data 15 day forecast data products
+#'
+#' @author Swarnalee Mazumder
+#'
+
+library(reticulate)
```

Comment:
please move library calls to `DESCRIPTION` file, e.g. see [here](https://github.com/PecanProject/pecan/blob/develop/modules/data.remote/DESCRIPTION#L26). I guess in this case you'll need to move `reticulate` from `Suggests` to `Imports` for `data.atmosphere`